repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -194,25 +194,26 @@ export const getServicesByNocCodeAndDataSourceWithGrouping = async (
     }
 };
 
-export const getFerryAndBusServices = async (noc: string): Promise<ServiceWithOriginAndDestination[]> => {
```

Comment:
I think this DB call needs to go back to how it was, and a new one be created. However, the name getFerryAndBusServices needs to be getFerryAndTramServices instead.